### PR TITLE
Spelling build

### PR DESCRIPTION
**Pull request by jsoref**
##### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

##### Component Name

> [stuff](https://github.com/netdata/netdata/pull/10428#issuecomment-760294967) that @Ferroin is responsible for

##### Test Plan

The misspellings have been reported at https://github.com/jsoref/netdata/commit/33c5b20ed9511dac0a23ee916fa247cc2f388304#commitcomment-45444734

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/netdata/commit/e2737c090e4b1d7b1bce405b9f702bcf526852a3

##### Additional Information

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: web/api/netdata-swagger.yaml
Comment (jsoref) [inline web/api/netdata-swagger.yaml:455]: Review dog may hate this, but it's a pre-existing condition.
yaml files do support a line wrapping thing, and wrapping this file (or similar) could be done in a distinct PR.

File: daemon/system-info.sh
Comment (jsoref) [inline daemon/system-info.sh:225]: This is scary

File: packaging/installer/install-required-packages.sh
Comment (jsoref) [inline packaging/installer/install-required-packages.sh:326]: CentOS is also misspelled

File: packaging/installer/kickstart-static64.sh
Comment (jsoref) [inline packaging/installer/kickstart-static64.sh:274]: This is scary

File: packaging/makeself/install-or-update.sh
Comment (jsoref) [inline packaging/makeself/install-or-update.sh:61]: scary

File: collectors/python.d.plugin/varnish/varnish.chart.py
Comment (jsoref) [inline collectors/python.d.plugin/varnish/varnish.chart.py:106]: ?

File: collectors/python.d.plugin/proxysql/proxysql.chart.py
Comment (jsoref) [inline collectors/python.d.plugin/proxysql/proxysql.chart.py:293]: ?

File: collectors/python.d.plugin/openldap/openldap.chart.py
Comment (jsoref) [inline collectors/python.d.plugin/openldap/openldap.chart.py:53]: If this is an upstream API, this could break

File: collectors/python.d.plugin/elasticsearch/elasticsearch.chart.py
Comment (jsoref) [inline collectors/python.d.plugin/elasticsearch/elasticsearch.chart.py:248]: Hopefully these are rendering

File: web/api/badges/README.md
Comment (jsoref) [inline web/api/badges/README.md:193]: The `|`s don't line up...

File: packaging/installer/methods/packages.md
Comment (jsoref) [inline packaging/installer/methods/packages.md:3]: While this is in `<!--`, it's possible this metadata change may impact something...

File: packaging/installer/UPDATE.md
Comment (jsoref) [inline packaging/installer/UPDATE.md:29]: This is a kind of important user facing change as the current text is misleading

File: HISTORICAL_CHANGELOG.md
Comment (jsoref) [inline HISTORICAL_CHANGELOG.md:385]: ?

File: HISTORICAL_CHANGELOG.md
Comment (jsoref) [inline HISTORICAL_CHANGELOG.md:486]: ?

File: backends/opentsdb/README.md
Comment (jsoref) [inline backends/opentsdb/README.md:26]: This seems like a correct change in context (the changes were done in a fairly automated manner)

File: collectors/node.d.plugin/named/named.node.js
Comment (jsoref) [inline collectors/node.d.plugin/named/named.node.js:325]: This is a bit scary

File: collectors/all.h
Comment (jsoref) [inline collectors/all.h:283]: I'm hoping these are considered private (yes, I fixed the whitespace here before submitting)

File: web/gui/dashboard.css
Comment (jsoref) [inline web/gui/dashboard.css:1]: I hope this is a typo

File: health/notifications/health_alarm_notify.conf
Comment (jsoref) [inline health/notifications/health_alarm_notify.conf:947]: This change is kinda scary

File: health/notifications/health_alarm_notify.conf
Comment (jsoref) [inline health/notifications/health_alarm_notify.conf:1178]: As is this one

File: collectors/statsd.plugin/statsd.c
Comment (jsoref) [inline collectors/statsd.plugin/statsd.c:1343]: whitespace (sigh)

File: collectors/perf.plugin/perf_plugin.c
Comment (jsoref) [inline collectors/perf.plugin/perf_plugin.c:1234]: whitespace

File: collectors/freebsd.plugin/freebsd_sysctl.c
Comment (jsoref) [inline collectors/freebsd.plugin/freebsd_sysctl.c:3019]: whitespace; ff.

**Review by jsoref — COMMENTED** [review]
In general, I expect to be asked to reorganize my changes (possibly into multiple PRs, possibly by file location or by file type, or by change type). I try to avoid squashing until the changes are generally acceptable as it's easiest to rebase (or drop changes) while they're distinct.

Overall, the CI tooling doesn't appear to particularly hate these changes.

I've left some review notes -- some are mostly to myself (I may address them before reviews) -- the rest are to call attention to things.

Do note that GitHub hates PRs w/ >250 commits and this PR is just a tad over the limit. You'll want to either review locally or suggest a partition of the PR.

File: web/api/netdata-swagger.yaml
Comment (jsoref) [reply web/api/netdata-swagger.yaml:455]: #10495

**Comment by ilyam8** [comment]
@jsoref thx for the PR, very nice job 👍 There are a lot of misspellings 😄 

This PR touches a lot of files, it won't be easy to get it reviewed and merged soon. What is easy is to fix all the `md` files.

Is it much to to create a separate PR that fixes only `md` files misspellings?

File: daemon/system-info.sh
Comment (Ferroin) [reply daemon/system-info.sh:225]: Luckily this is non-critical code. The misspelling got missed in testing, and the error message it (should) generate is not likely seen by anybody because of how the script is actually used.

**Comment by Ferroin** [comment]
> @jsoref thx for the PR, very nice job  There are a lot of misspellings 
> 
> This PR touches a lot of files, it won't be easy to get it reviewed and merged soon. What is easy is to fix all the `md` files.
> 
> Is it much work to to create a separate PR that fixes only `md` files misspellings?

Same request actually for the stuff under `.travis`, `health/notifications`, and `packaging` and all of the `.sh` and `.sh.in` files. All of _that_ stuff specifically is relatively trivial to review, and those include fixes for a couple of bugs resulting from misspellings.

**Comment by jsoref** [comment]
Splitting out files (by path or file extension) is fairly trivial (I do this regularly, my tooling doesn't mind -- the only thing that you'll see is if a misspelling crosses a boundary I may drop it and do it separately)

It'd be helpful if there was a bulleted list of the items -- I can't tell if:
- .travis, health/notifications, and packaging and all of the .sh and .sh.in files. -- is a single PR or 3 separate -- I suspect it's one for "build stuff"

I can do one for md first though since I understand that :-)

**Comment by Ferroin** [comment]
> * .travis, health/notifications, and packaging and all of the .sh and .sh.in files. -- is a single PR or 3 separate -- I suspect it's one for "build stuff"

Strictly speaking, it’s ‘stuff that I (@Ferroin) am responsible for and have already confirmed are the changes are correct for’.

In practice, most of it is build and packaging stuff, but the main reason i care is that fixing spelling in a handful of these places actually fixes (thankfully non-critical) bugs we appear to have had for quite some time.

If it’s easier for you to just do it by file extensions though that’s fine too.

**Comment by jsoref** [comment]
Sure. I'm not trying to be difficult. It sounds like it's a single PR for all of those things (minus the PR for `.md` files that I've already split out).

That's totally manageable. Hopefully tonight. If not perhaps Sunday.

File: database/rrddim.c
Comment (vlvkobal) [inline database/rrddim.c:197]: The name of the variable is not very good but it shouldn't be `username` anyway.

File: collectors/freebsd.plugin/freebsd_sysctl.c
Comment (vlvkobal) [inline collectors/freebsd.plugin/freebsd_sysctl.c:2996]: I would leave the variables as they were. InParmProblems and OutParmProblems are consistent with the proc plugin names, the variables were named after them.

File: database/rrddim.c
Comment (jsoref) [reply database/rrddim.c:197]: Fwiw, I've run into `username` typo'd as [usename](https://www.postgresql.org/docs/9.3/view-pg-user.html) in the past. -- That wasn't fixable because it's an ancient published mistake that things rely on (as w/ http's [Referer](https://lists.w3.org/Archives/Public/ietf-http-wg-old/1995JanApr/0107.html)

Would `use_name` be ok? -- That seems to be the general convention here.

File: collectors/freebsd.plugin/freebsd_sysctl.c
Comment (ilyam8) [reply collectors/freebsd.plugin/freebsd_sysctl.c:2996]: https://iphostmonitor.com/mib/oids/ibm-icmpv6/ipv6IfIcmpInParmProblems.html

Comment (jsoref) [reply collectors/freebsd.plugin/freebsd_sysctl.c:2996]: Yeah, I consciously left the upstream object alone. The :eyes: are a mark reminding me to fix a thing.

File: database/rrddim.c
Comment (vlvkobal) [reply database/rrddim.c:197]: It's a local variable with a very small scope. I think it can be simply `name`. @thiagoftsm did you have any special meaning in mind when you named the variable `usename`?

Comment (thiagoftsm) [reply database/rrddim.c:197]: You are right @vlvkobal  we can change for `name`, the initial intention was to say that the dimension needs `to use the name`.

**Comment by jsoref** [comment]
@Ferroin: as long as there's a rule I can use to delete files, it doesn't really matter -- basically, I branch master, delete things that you don't want, rebase onto that branch, delete conflicts w/ deleted files, continue until done rebasing, rebase onto master dropping the delete commit.

This PR should now cover the things you described.

The other PR is the result of rebasing the unfiltered branch back onto the filtered branch, and then rebasing back to master dropping the filtered commits.

(BTW, the gift arrived, thanks)

**Review by Ferroin — APPROVED** [review]
There are exactly four cases I can see where this changes behavior, all of which fix bugs in the existing code.

All the other changes are purely cosmetic, but semantically correct.

**Comment by Ferroin** [comment]
@jsoref The checksum check will need to be fixed before this can be merged, but I can take care of that if you want.

If you want to just handle it yourself, the relevant changes are to update the MD5 checksums in `packaging/installer/methods/kickstart*.md` to match the new MD5 checksums for the respective kickstart shell scripts in `packaging/installer`.

**Comment by jsoref** [comment]
I'd rather leave that to you, if you don't mind.

**Comment by Ferroin** [comment]
> I'd rather leave that to you, if you don't mind.

OK, I’ll push a commit to the branch then to update the checksums (probably some time early tomorrow).

**Review by Ferroin — DISMISSED** [review]

**Comment by jsoref** [comment]
I'm leaving this to you folks, but https://github.com/netdata/netdata/pull/10428/checks?check_run_id=1735278628 would be frustrating if I was responsible for pushing it along :-/

**Review by vlvkobal — DISMISSED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Comment by ilyam8** [comment]
@jsoref hi, sorry for the delay. Can you fix the conflicts so we can merge it?

**Comment by jsoref** [comment]
This dropped c28b0509366650eaa53a1bbc298f1a203acc257f because I hit conflicts. if @Ferroin or someone else could repeat that step and then merge, that'd be great.

**Review by vlvkobal — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**
